Widget2

wesj opened the pull request.

Commit:
Add branding to grunt

Commit:
Add branding to grunt

Commit:
Add gecko resources

Commit:
Add gecko resources

Commit:
Add resources

Commit:
Add resources

Commit:
Make this build

Commit:
Make this build

Commit:
Add nightly preview image

Commit:
Add nightly preview image

Commit:
Don't push icon back into Fennec

Commit:
Don't push icon back into Fennec

wesj commented:
I think some things like strings and colors that I'm stealing from Fennec shouldn't be in here. Not sure how we handle that.

EricEdens commented:
> I think some things like strings and colors that I'm stealing from Fennec
> shouldn't be in here. Not sure how we handle that.
> 
> We've copied a few things from MC so that the project can compile -- for
> example BrowserContract. During the export process (_grunt export --tree .._)
> we ignore those copied resources.

In the case of strings and colors, I expect we'd create these resources in
their own file (eg: gecko_strings), and then ignore that when doing the
export.

I can help with updating our grunt config in order to ignore those
resources.

Commit:
Remove branding changes. I landed those in central themselves.

Commit:
Remove branding changes. I landed those in central themselves.

Commit:
Update app constants and class name

Commit:
Update app constants and class name

Commit:
Merge branch 'widget2' of https://github.com/wesj/FirefoxSearch into widget2

Conflicts:
	app/src/main/res/values/search_dimens.xml
	strings/search_strings.dtd
	strings/search_strings.xml.in

wesj commented:
I updated this to get rid of the branding dir entirely. I landed those changes in fx-team independent of this. I also updated AppConstants to include/use the new code.

wesj commented:
Testing grunt export this patch seems to work fine on central. I think we end up with some duplicated color values, but I'm willing to live with that for now.

leibovic commented on app/src/main/res/values/search_dimens.xml:
```diff
@@ -23,4 +23,18 @@
+    <dimen name="widget_button_offset">-50dp</dimen>
+    <dimen name="widget_button_padding">45dp</dimen>
+    <dimen name="widget_text_size">15sp</dimen>
+    <dimen name="widget_drawable_padding">2dp</dimen>
```
You accidentally declared all these values twice. You should just declare them once :)

leibovic commented:
Closing this, since this patch was merged in a different PR.
